manifests/charts/default/values.yaml

```diff
@@ -14,3 +14,8 @@ istiodRemote:
 
 # Revision is set as 'version' label and part of the resource names when installing multiple control planes.
 revision: ""
+
+sidecarInjectorWebhook:
```

Comment:
The default chart is rendered through code with [hardcoded values](https://github.com/istio/istio/blob/master/istioctl/pkg/tag/generate.go#L208), so this won't get set (even if setting `sidecarInjectorWebhook.enableNamespacesByDefault=true` on the pilot chart). I think we need to add another [generate option](https://github.com/istio/istio/blob/master/istioctl/pkg/tag/generate.go#L56) with a new flag to `istioctl tag` for this.

Reply:
Thanks for explaining, I have added the generate option, PTAL